modules/nf-neuro/segmentation/synthseg/meta.yml

```diff
@@ -0,0 +1,84 @@
+name: "segmentation_synthseg"
+description: Perform Brain Tissues Segmentation using Freesurfer synthseg on a T1 image. Optionally, a binary mask of lesion can be add to correct the white matter mask.
+keywords:
+  - Segmentation
```

Comment:
Add keywords to all ML/AI modules, somethings like `synthetic`, `Machine Learning`, `Artificial Intelligence`, `Neural Network`, maybe the type of network or model used ...